### PR TITLE
Fix/tabs width

### PR DESCRIPTION
**Pull request by Z-AOAO**
## 📝 Description

Add a brief description.

## 💣 Is this a breaking change (Yes/No):

- [ ] Yes
- [x] No

## 🚧 How to migrate?

Don't need.

## 📝 Additional Information

lol

### CODE REVIEW

**Review by zch-modier — APPROVED** [review]

File: packages/tabs/src/headers/tab-common-header.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ export const TabCommonHeader = forwardRef<HTMLDivElement, TabHeaderProps>(
 
     return (
       <div css={applyHeaderContainerCss(false)} ref={ref}>
-        {prefix}
+        <React.Fragment key="prefix">{prefix}</React.Fragment>
```
Comment (xiaoyu-a) [inline packages/tabs/src/headers/tab-common-header.tsx:101]: 这里不要引React，引Fragment

File: packages/tabs/src/headers/tab-line-header.tsx [hunk 2]
```diff
@@ -199,7 +184,7 @@ export const TabLineHeader = forwardRef<HTMLDivElement, TabHeaderProps>(
         css={applyLineHeaderContainerCss(_isHorizontalLayout, tabPosition)}
         ref={ref}
       >
-        {prefix}
+        <React.Fragment key="prefix">{prefix}</React.Fragment>
```
Comment (xiaoyu-a) [inline packages/tabs/src/headers/tab-line-header.tsx:187]: 同上

File: packages/tabs/src/headers/tab-line-header.tsx [hunk 3]
```diff
@@ -288,7 +269,7 @@ export const TabLineHeader = forwardRef<HTMLDivElement, TabHeaderProps>(
             </span>
           )}
         </div>
-        {suffix}
+        <React.Fragment key="suffix">{suffix}</React.Fragment>
```
Comment (xiaoyu-a) [inline packages/tabs/src/headers/tab-line-header.tsx:272]: 同上

File: packages/tabs/src/headers/tab-common-header.tsx [hunk 4]
```diff
@@ -5,6 +5,7 @@ import React, {
   useMemo,
```
Comment (xiaoyu-a) [inline packages/tabs/src/headers/tab-common-header.tsx:5]: 上面的import React 要去掉

**Review by xiaoyu-a — APPROVED** [review]

File: packages/tabs/src/headers/tab-line-header.tsx [hunk 5]
```diff
@@ -5,11 +5,10 @@ import React, {
   useMemo,
```
Comment (xiaoyu-a) [inline packages/tabs/src/headers/tab-line-header.tsx:5]: 同上

**Pull request closed without merging**